bin/deploy-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
+set -eo pipefail
+
+# Strip the 'v' from the Tag Name
+export TAG=${TAG_NAME:1}
+
+# Deploy release to OSSRH
+# 1. Set the version in the POM to the Tagged Version
+# 2. Sign our build and deploy to OSSRH
+# 3. Release our staged deployment
+# Note: The autoReleaseAfterClose for the nexus-staging-maven-plugin should be
+# set to "false" if we do not want releases published automatically
+docker run --rm \
+  -e OSSRH_USERNAME \
+  -e OSSRH_PASSWORD \
+  -v "$PWD:/conjurinc/api-java" \
+  -v "$GPG_PASSWORD:/gpg_password" \
+  -v "$GPG_PRIVATE_KEY:/gpg_key" \
+  -w /conjurinc/api-java maven:3-jdk-8 \
+  /bin/bash -c "gpg --batch --passphrase-file /gpg_password --trust-model always --import /gpg_key &&
```

Comment:
```suggestion
  /bin/bash -ec "gpg --batch --passphrase-file /gpg_password --trust-model always --import /gpg_key &&
```

Reply:
With this change, you don't need the `&&`s I think

Reply:
![image](https://user-images.githubusercontent.com/19347386/85303750-e6d4c380-b478-11ea-977d-0d0630e63100.png)